src/components/PostDetails.tsx

```diff
@@ -2,92 +2,31 @@ import React, { useEffect, useState } from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
 
-import * as commentsApi from '../api/comments';
-
 import { Post } from '../types/Post';
-import { Comment, CommentData } from '../types/Comment';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
+import {
+  deleteComment,
+  loadPostComments,
+} from '../features/comments/сommentsSlice';
```

Comment:
There is a typo in the import path for `deleteComment` and `loadPostComments`. The directory name `сommentsSlice` seems to be misspelled. Ensure that the path is correct to avoid import errors.